lib/matplotlib/tests/test_triangulation.py

```diff
@@ -730,7 +730,7 @@ def z(x, y):
             matest.assert_array_almost_equal(interpz, interp_z0[interp_key])
 
 
-@image_comparison(['tri_smooth_contouring.png'], remove_text=True, tol=0.07)
+@image_comparison(['tri_smooth_contouring.png'], remove_text=True, tol=0.072)
```

Comment:
It is probably worth trying the PR without these tolerance changes.  They were needed for 2019.1 but may not be for 2020.2.

Reply:
they're still needed, I have checked.

Reply:
Ok, thanks for checking.